src/main/java/uk/gov/hmcts/reform/sscs/validators/FormTypeValidator.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.reform.sscs.validators;
+
+import static uk.gov.hmcts.reform.sscs.constants.SscsConstants.FORM_TYPE;
+import static uk.gov.hmcts.reform.sscs.helper.OcrDataBuilder.build;
+import static uk.gov.hmcts.reform.sscs.helper.SscsDataHelper.getValidationStatus;
+import static uk.gov.hmcts.reform.sscs.util.SscsOcrDataUtil.getField;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.bulkscancore.domain.CaseResponse;
+import uk.gov.hmcts.reform.sscs.bulkscancore.domain.ExceptionRecord;
+import uk.gov.hmcts.reform.sscs.bulkscancore.domain.ScannedData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.FormType;
+import uk.gov.hmcts.reform.sscs.json.SscsJsonExtractor;
+
+
+
+@Component
+@Slf4j
+public class FormTypeValidator {
+
+    private final SscsJsonExtractor sscsJsonExtractor;
+    private final Schema sscs1Schema = tryLoadSscsSchema("/schema/sscs-bulk-scan-schema.json");
+    private final Schema sscs2Schema = tryLoadSscsSchema("/schema/sscs2-bulk-scan-schema.json");
+    private final Schema sscs5Schema = tryLoadSscsSchema("/schema/sscs5-bulk-scan-schema.json");
+
+    public FormTypeValidator(SscsJsonExtractor sscsJsonExtractor) {
+        this.sscsJsonExtractor = sscsJsonExtractor;
+    }
+
+    public CaseResponse validate(String caseId, ExceptionRecord exceptionRecord) {
+        List<String> errors = null;
+
+        String formTypeData = getFormType(caseId, exceptionRecord);
+
+        if (formTypeData == null) {
+            errors = new ArrayList<>();
+            errors.add("No valid form type was found, need to add form_type with valid form type to OCR data");
+            log.info("No valid form type was found while transforming exception record caseId {}",
+                caseId);
```

Comment:
I would probably return the errors now to avoid having to nest the else statement in a block